wikimedia/wikipedia-ios
# Manually check for Farsi when searching for languages

**Phabricator:** https://phabricator.wikimedia.org/T107530

### Notes
* Quick hack to check for Farsi in search box, per discussion.
* Also refactored out some duplicated code.

### Test Steps
1. Run app, go to adding languages.
2. Search for "Farsi". After every letter typed, "Persian" should be an option as well as the expected results.
3. "Persian" should also be available when searching for "Persian", of course.
